Asset viewers and animation previews (#494)
playcanvas/developer.playcanvas.com: content/en/user-manual/assets/animation.md

An Animation asset is used to play a single animation on a 3D model. Animations are imported by uploading 3D scenes (such as FBX files) which contain animation data. The animation data is extracted from the uploaded file by the [asset pipeline][asset_pipeline] and a [Target Asset][target_asset] is created to use in game.

## Animation Preview

The inspector has a viewer that can be used to preview the animation with any Template or Model asset. This is useful if you have a single animation that can be applied to different character rigs.

> ![Asset Inspector Preview][inspector-preview-gif]

Reviewer: I know we've discussed this before, but I'm thinking we should enforce the use of number-based references. This `inspector-preview-gif` string, for example, appears twice in the document, which is not much different (in terms of characters) to just embedding the link directly.

One other thing that is maybe less obvious, is that these references appear in the Transifex interface. Some translators might be attempted to translate the reference - not so with a number.

So right now, we have a seemingly worsening divergence between the use of numbers and strings....so I think we should just universally switch back to numbers. Not in this PR, but in a follow up somewhere down the line.

Author: As a third option, how do you feel about inlining links instead of references? I generally find that I'm more error prone with number based references when editing a lot of these docs hence switching to text based.

But if we inline (it's not often that we use the same link twice), that saves a lot of back of forth in the document when trying to write these docs

Reviewer: I don't like inlining links because it pollutes paragraphs that make life difficult especially for translators in Transifex.

In academic papers, for example, references are generally done with superscript numbers. To me, they feel much more natural.

Author: Okay, I keep this in mind with future PRs and switch back to numbered references

> There is also a viewer in the [Anim State Graph Editor][anim-state-graph-editor] when you select a state in the graph. It will show all Entities in the scene using the graph where you can select one to preview.

Reviewer: That last sentence is kinda hard to understand. Could it be rephrased somehow to make it clearer?

Author: ```suggestion
There is also a viewer in the [Anim State Graph Editor][anim-state-graph-editor] when you select a state in the graph. When selected, all Entities in the scene that are using the graph will show in a list and can be selected for preview.
```

How's this?

Reviewer: Much better. 👍
